tol_lab_share/messages/output_traction_message.py

```diff
@@ -0,0 +1,190 @@
+from typing import Dict, Optional, List
+from json import dumps
+from requests import post, codes
+from tol_lab_share.messages.output_feedback_message import OutputFeedbackMessage
+from tol_lab_share.messages.message import Message
+
+from tol_lab_share import error_codes
+from tol_lab_share.error_codes import ErrorCode
+from requests.exceptions import JSONDecodeError
+
+
+class OutputTractionMessageRequest:
+    def __init__(self):
+        self._library_type = None
+        self._study_uuid = None
+        self._sample_name = None
+        self._sample_uuid = None
+        self._container_barcode = None
+        self._container_location = None
+        self._container_type = None
+        self._species = None
+
+    @property
+    def species(self) -> Optional[str]:
+        return self._species
+
+    @species.setter
+    def species(self, value: Optional[str]) -> None:
+        self._species = value
+
+    @property
+    def container_type(self) -> Optional[str]:
+        return self._container_type
+
+    @container_type.setter
+    def container_type(self, value: Optional[str]) -> None:
+        self._container_type = value
+
+    @property
+    def container_location(self) -> Optional[str]:
+        return self._container_location
+
+    @container_location.setter
+    def container_location(self, value: Optional[str]) -> None:
+        self._container_location = value
+
+    @property
+    def container_barcode(self) -> Optional[str]:
+        return self._container_barcode
+
+    @container_barcode.setter
+    def container_barcode(self, value: Optional[str]) -> None:
+        self._container_barcode = value
+
+    @property
+    def sample_uuid(self) -> Optional[str]:
+        return self._sample_uuid
+
+    @sample_uuid.setter
+    def sample_uuid(self, value: Optional[str]) -> None:
+        self._sample_uuid = value
+
+    @property
+    def sample_name(self) -> Optional[str]:
+        return self._sample_name
+
+    @sample_name.setter
+    def sample_name(self, value: Optional[str]) -> None:
+        self._sample_name = value
+
+    @property
+    def library_type(self) -> Optional[str]:
+        return self._library_type
+
+    @library_type.setter
+    def library_type(self, value: Optional[str]) -> None:
+        self._library_type = value
+
+    @property
+    def study_uuid(self) -> Optional[str]:
+        return self._study_uuid
+
+    @study_uuid.setter
+    def study_uuid(self, value: Optional[str]) -> None:
+        self._study_uuid = value
+
+    def serializer(self):
+        return RequestSerializer(self)
+
+
+class RequestSerializer:
+    # "cost_code": "0000", "data_type": "basecalls"
```

Comment:
Not sure what this is. What are we doing with regard to ONT?

Reply:
If it has library type ont it will send different message. Updated